examples/21_create-pages/src/entries.tsx

```diff
@@ -140,6 +140,16 @@ const pages = createPages(
       },
     }),
 
+    createApi({
+      path: '/api/hi/[name]',
+      method: 'POST',
```

Comment:
I think with `POST` we want to have an example with reading the HTTP request body.

Reply:
changed!